### ISSUE TITLE
Sparta 

### ISSUE THREAD

**Issue by tanvipatel123** [tanvipatel123/js-tanvi#1]
<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
    crossorigin="anonymous"></script>
    <title>Fan Book Site</title>
    <link href="https://fonts.googleapis.com/css2?family=Gowun+Dodum&display=swap" rel="stylesheet">

    <style>
      .mypic{
        width: 100%;
        height: 300px;
        background-image:linear-gradient(0deg, rgba(0,0,0,0.5),rgba(0,0,0,0.5)),url('bts.jpg') ;
        background-position: center;
        background-size: cover;
        color: white;
        display: flex;
        flex-direction: column;
        align-items: center;
        justify-content: center;
      }
      .mypost{
        width: 95%;
        max-width: 500px;
        margin: 20px auto 20px auto;
        padding: 20px;
        box-shadow: 0px 0px 3px 0px black;
      }
      .mypost>button{
        margin-top: 15px;
      }
      .mycards{
        width: 95%;
        max-width: 500px;
        margin: auto;
      }
      .mycards > .card{
        margin-top: 10px;
        margin-bottom: 10px;
      }
  </style>
</head>

<body>
    <div class="mypic">
        <h1>BTS Fan Book</h1>
    </div>
      <div class="mypost">
        <div class="form-floating mb-3">
          <input type="text" class="form-control" id="floatingInput" placeholder="Enter Your Name">
          <label for="floatingInput" class="form-label">Nickname</label>
        </div>
        <div class="form-floating">
          
          <textarea class="form-control" id="form-label" rows="3" placeholder="Leave comment here"></textarea>
          <label for="floatingTextarea" class="form-label">Fan Message</label>
        </div>
        <button type="button" class="btn btn-dark">Fan Message</button>

      </div>
      <div class="mycards">
        <div class="card">
          <div class="card-body">
            <blockquote class="blockquote mb-0">
              <p>A well-known quote, contained in a blockquote element.</p>
              <footer class="blockquote-footer">Someone famous in <cite title="Source Title">Source Title</cite></footer>
            </blockquote>
          </div>
        </div>

        <div class="card">
          <div class="card-body">
            <blockquote class="blockquote mb-0">
              <p>A well-known quote, contained in a blockquote element.</p>
              <footer class="blockquote-footer">Someone famous in <cite title="Source Title">Source Title</cite></footer>
            </blockquote>
          </div>
        </div>

        <div class="card">
          <div class="card-body">
            <blockquote class="blockquote mb-0">
              <p>A well-known quote, contained in a blockquote element.</p>
              <footer class="blockquote-footer">Someone famous in <cite title="Source Title">Source Title</cite></footer>
            </blockquote>
          </div>
        </div>
      </div>
    
</body>

</html>
